enterprise/dbcrypt/dbcrypt_internal_test.go

```diff
@@ -349,6 +349,112 @@ func TestExternalAuthLinks(t *testing.T) {
 	})
 }
 
+func TestCryptoKeys(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	// We don't write a GetCryptoKeyByFeatureAndSequence test
+	// because it's basically the same as InsertCryptoKey.
```

Comment:
This comment doesn't make sense.  You _do_ have a GetCryptoKeyByFeatureAndSequence test